src/super_gradients/common/environment/omegaconf_utils.py

```diff
@@ -36,3 +36,4 @@ def register_hydra_resolvers():
     OmegaConf.register_new_resolver("getitem", lambda container, key: container[key], replace=True)  # get item from a container (list, dict...)
     OmegaConf.register_new_resolver("first", lambda lst: lst[0], replace=True)  # get the first item from a list
     OmegaConf.register_new_resolver("last", lambda lst: lst[-1], replace=True)  # get the last item from a list
+    OmegaConf.register_new_resolver("choose", lambda key, options: options[key], replace=True)
```

Comment:
What is the difference of this resolver to getitem? The order of container/key is swapped but apart from this they are the same

Reply:
Apparently, I missed it.
[77ac604](https://github.com/Deci-AI/super-gradients/pull/594/commits/77ac60477c40fd249040c61a185d8fcb49c6e25d)